data/human/free worlds 1 start.txt

```diff
@@ -1266,7 +1266,7 @@ mission "FW Pirates: Attack 1"
 			label senate
 			`	JJ puts a hand on his shoulder and says, a little more firmly than Alondo did, "Calm down, Tomek. We're obeying the orders of the Senate. We have to honor the democratic process even when we don't like the decisions they make."`
 			`	"Oh, the Senate. Of course. The Senate this, the Senate that," Tomek says mockingly while throwing his head back. "The Senate is in no position to be commanding this war. They do not have the military experience that I have, or that you have JJ. Do you think the Republic Parliament is busying themselves voting on every little fleet movement, or do you think they've put this in the hands of the Navy and told them to win this war? I made my reasons clear to you all many times why I didn't want to convene the Senate, because they would only hinder and handicap us from winning this war, and that's exactly what they're already doing!"`
-			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council was decided that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
+			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council decided that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
```

Comment:
```suggestion
			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council was clear that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
```
Removing the word "decided" and replacing it with a synonym should help prevent the confusion. To me these words have the same meaning, but I don't know about other people. I do think that this is a better idea than trying to choose between keeping it as an adjective, making the text confusing, or making it a verb, changing the meeting.

Reply:
```suggestion
			`	"Tomek! That's enough!" JJ yells, which seems to draw even more surprise and attention than when Tomek yelled. "The Council has decidedly determined that the Senate would convene, and so it did. It is only by the grace of the Senate that the Council even still exists."`
```
Alternative suggestion: This changes the meaning of the text, in order to show that the Council does not regret their decision, while still showing that the Council was clear when they made the decision. If the goal is to preserve the initial meaning, my initial suggestion ensures that people will be less tripped up by the (correct) grammar.